plugins/DataLoadRlog/rlog_parser.cpp

```diff
@@ -78,3 +88,30 @@ bool RlogMessageParser::parseMessageImpl(const std::string& topic_name, capnp::D
   }
   return true;
 }
+
+bool RlogMessageParser::parseCanMessage(
+  const std::string& topic_name, capnp::DynamicList::Reader listValue, double time_stamp) 
+{
+  if (dbc_name.empty()) {
+    return false;
+  }
+  std::vector<uint8_t> updated_busses;
```

Comment:
I would use `std::set<uint8_t>` so you process every bus only once in the next step.